src/main/java/org/synyx/urlaubsverwaltung/overtime/OvertimeSettings.java

```diff
@@ -31,6 +31,14 @@
     @Column(name = "overtime_minimum")
     private Integer minimumOvertime = 5;
 
+    /**
+     * Defines the minimum overtime reduction value a person has to use.
```

Comment:
```suggestion
     * Defines the minimum overtime reduction value a person has to use for an application.
```
Es ist ja prinzipiell noch möglich die Überstunden für "Überstunden abbauen statt hinzufügen"-Checkbox abzubauen.